src/Dofs/ConstraintHandler.jl

```diff
@@ -386,68 +393,27 @@ function meandiag(K::AbstractMatrix)
 end
 
 
-
-# TODO Remove - almost identical to the previous
-function _update!(values::Vector{Float64}, f::Function, faces::Set{Tuple{Int,Int}}, field::Symbol, local_face_dofs::Vector{Int}, local_face_dofs_offset::Vector{Int},
```

Comment:
Delete this function anduse the normal _update! function for both MixedDofHandler and DofHandler